src/js/render.js

```diff
@@ -0,0 +1,104 @@
+import { local, getBackground } from './helpers';
+
+const DOM = {
+  mainBackground: document.querySelector('.main-image'),
+  sideBackground: document.querySelector('.side-image'),
+  forecast: document.querySelector('.forecast'),
+  switchTemp: document.querySelector('.switchTemp'),
+  celcius: document.querySelector('.celcius'),
+  farenheit: document.querySelector('.farenheit'),
+  searchBar: document.getElementById('search-bar'),
+};
+
+const renderPage = () => {
+  DOM.forecast.classList.add('hidden');
+  DOM.mainBackground.src = getBackground('default');
+  DOM.sideBackground.src = getBackground('default');
+};
+
+const renderSearch = (currentData, forecastData) => {
+  const degSymbol = '°';
+  const description = document.querySelector('.description');
+  const city = document.querySelector('.city');
+  const time = document.querySelector('.time');
+  const date = document.querySelector('.date');
+  const icon = document.querySelector('.icon');
+  const switchTemp = document.querySelector('.switchTemp');
+  DOM.mainBackground.src = getBackground(currentData.code).toString();
+  DOM.sideBackground.src = getBackground(currentData.code).toString();
+
+  const main = () => {
+    DOM.celcius.innerHTML = `${currentData.cTemp.toFixed()}${degSymbol} `;
+    DOM.farenheit.innerHTML = `${currentData.fTemp.toFixed()}${degSymbol} `;
+    description.innerHTML = currentData.description;
+    city.innerHTML = currentData.city;
+    const dateFn = new Date(`${currentData.date} UTC`);
+    const weekDay = local.days[new Date(`${currentData.date} UTC`).getDay()];
+    const month = local.months[new Date(`${currentData.date} UTC`).getMonth()];
+    const day = new Date(`${currentData.date} UTC`).getDay();
+    const hour = dateFn.getHours();
+    const minutes = (dateFn.getMinutes() < 10 ? '0' : '') + dateFn.getMinutes();
+    time.innerHTML = `${hour}: ${minutes} `;
+    date.innerHTML = `${weekDay}, ${month} ${day} `;
+    icon.src = `http://openweathermap.org/img/wn/${currentData.icon}@2x.png`;
+  };
+
+  const aside = () => {
+    const switchBtn = document.createElement('button');
+    switchBtn.classList.add('switch');
+    switchBtn.innerHTML = '°F';
+    switchTemp.innerHTML = '';
+    switchTemp.append(switchBtn);
+    DOM.forecast.classList.remove('hidden');
+    const list = document.querySelector('.forecast-list');
+    list.innerHTML = '';
+
+    Object.keys(forecastData).forEach((day) => {
```

Comment:
Seems like you could use `Object.values(forecastData).forEach((day) => {`, so then you can replace `${forecastData[day].date}` with `${day.date}`

Reply:
was causing too much trouble so I left it like that